resources/migrations/000_migrations.yaml

```diff
@@ -14362,6 +14361,62 @@ databaseChangeLog:
               WHEN MATCHED THEN
                 UPDATE SET f.base_type = updates.base_type;
 
+  - changeSet:
+      id: v47.00-002
+      author: calherries
+      comment: Added 0.47.0 - Add json_unfolding column to metabase_field
+      changes:
+        - addColumn:
+            columns:
+            - column:
+                remarks: 'Enable/disable JSON unfolding for a field'
+                name: json_unfolding
+                type: boolean
+                constraints:
+                  nullable: true
```

Comment:
should we have another migration to make this non-nullable and have a default value?

FWIW I asked QP ppl about a similar problem for `database_requried` [here](https://metaboat.slack.com/archives/C04DN5VRQM6/p1678112076860729).

And the consensus is that we should have default value and make it nullable.

Reply:
True, json_unfolding could be `false` by default. Could prevent some footguns. Good idea